### ISSUE TITLE
Mixed line endings failure

### ISSUE THREAD

**Issue by oliveiraev** [pre-commit/pre-commit-hooks#237]
I'm facing the following traceback while trying to use mixed line endings v0.9.3:

```
Traceback (most recent call last):
  File "/home/evandro/.cache/pre-commit/repow2ta9av0/py_env-python3/bin/mixed-line-ending", line 11, in <module>
    load_entry_point('pre-commit-hooks==0.9.3', 'console_scripts', 'mixed-line-ending')()
  File "/home/evandro/.cache/pre-commit/repow2ta9av0/py_env-python3/lib/python3.5/site-packages/pkg_resources/__init__.py", line 565, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/evandro/.cache/pre-commit/repow2ta9av0/py_env-python3/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2631, in load_entry_point
    return ep.load()
  File "/home/evandro/.cache/pre-commit/repow2ta9av0/py_env-python3/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2291, in load
    return self.resolve()
  File "/home/evandro/.cache/pre-commit/repow2ta9av0/py_env-python3/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2301, in resolve
    raise ImportError(str(exc))
ImportError: module 'pre_commit_hooks.mixed_line_ending' has no attribute 'mixed_line_ending'

```

**Comment by asottile** [comment]
woops, I renamed the executable without fixing the entry_point, fix incoming!

**Comment by asottile** [comment]
This is now available in 0.9.4!

**Comment by oliveiraev** [comment]
thnx :+1:

Att, Evandro Oliveira.
